sonar-scala-plugin/src/main/resources/org/sonar/l10n/scala/rules/scalastyle/scalastyle-rules.json

```diff
@@ -0,0 +1,578 @@
+[
+  {
+    "key": "file.FileLengthChecker",
+    "name": "File size limit",
+    "description": "Files which are too long can be hard to read and understand.",
+    "type": "CODE_SMELL",
+    "severity": "MINOR",
```

Comment:
How do we decide what is CODE_SMELL/BUG or MINOR/MAJOR/...?

Reply:
```
Scapegoat Error   => BUG       /MINOR
Scapegoat Warning => CODE_SMELL/MAJOR
Scapegoat Info    => CODE_SMELL/MINOR
Scalastyle        => CODE_SMELL/MINOR
```